packages/backend/src/config/app.ts

```diff
@@ -58,7 +58,7 @@ export const Config = {
     ? Buffer.from(process.env.STREAM_SECRET, 'base64')
     : crypto.randomBytes(32),
 
-  useTigerbeetle: envBool('USE_TIGERBEETLE', true),
+  useTigerbeetle: envBool('USE_TIGERBEETLE', false),
```

Comment:
Can we remove the tigerbeetle testcontainer from the jest global setup?
https://github.com/interledger/rafiki/blob/d33d4d9b5c3423e7990d0151647062e78942d53b/packages/backend/jest.setup.ts#L74-L95